modules/repository/init.go

```diff
@@ -62,7 +62,8 @@ func prepareRepoCommit(ctx context.Context, repo *repo_model.Repository, tmpDir,
 	}
 	res, err := vars.Expand(string(data), match)
 	if err != nil {
-		return fmt.Errorf("expand README.md: %v", err)
+		// here we could just log the error and continue the rendering
+		log.Error("unable to expand template vars for repo README: %s, err: %s", opts.Readme, err.Error())
```

Comment:
Is that meaningful to write an empty to the file?

Reply:
No, it always has content. See the Expand unit test.